x-pack/legacy/plugins/siem/public/components/formatted_bytes/index.test.tsx

```diff
@@ -8,53 +8,56 @@ import { mount, shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import * as React from 'react';
 
-import { mockFrameworks, TestProviders } from '../../mock';
+import { useKibanaUiSetting } from '../../lib/settings/use_kibana_ui_setting';
+import { mockFrameworks, getMockKibanaUiSetting } from '../../mock';
 
 import { PreferenceFormattedBytes } from '.';
 
+const mockUseKibanaUiSetting: jest.Mock = useKibanaUiSetting as jest.Mock;
```

Comment:
Very nice in mocking this hook -- will use this method when testing going forward! 👍